sass/lightbox.scss

```diff
@@ -98,12 +121,6 @@ $button-padding: 40px;
       @include calc('top', '50% - 15px');
       position: absolute;
 
-      .lightbox-icon-bg {
-        opacity: .1;
-        fill: #26262A;
-        enable-background: new;
```

Comment:
Illustrator adds this line by default. But it's only for IE 10 and we don't need it.